pkg/actuators/machine/instances.go

```diff
@@ -509,53 +527,6 @@ func getInstanceMarketOptionsRequest(providerConfig *machinev1beta1.AWSMachinePr
 	return instanceMarketOptionsRequest
 }
 
-// constructInstancePlacement configures the placement options for the RunInstances request
-func constructInstancePlacement(machine *machinev1beta1.Machine, machineProviderConfig *machinev1beta1.AWSMachineProviderConfig, client runtimeclient.Client) (*ec2.Placement, error) {
```

Comment:
I would keep this function, but just remove the placement group specific bits, it's a nice refactor as it was

Reply:
👍